packages/amplify-category-api/package.json

```diff
@@ -30,12 +30,14 @@
   },
   "dependencies": {
     "@aws-amplify/graphql-auth-transformer": "2.1.12",
-    "@aws-amplify/graphql-schema-generator": "0.1.8",
+    "@aws-amplify/graphql-schema-generator": "0.2.0",
     "@aws-amplify/graphql-transformer": "0.0.7",
     "@aws-amplify/graphql-transformer-core": "1.3.8",
     "@aws-amplify/graphql-transformer-interfaces": "2.2.5",
     "@aws-amplify/graphql-transformer-migrator": "2.1.12",
-    "@aws-cdk/aws-apigatewayv2-alpha": "~2.80.0-alpha.0",
+    "@aws-cdk/aws-apigatewayv2-alpha": "~2.68.0-alpha.0",
```

Comment:
This shouldn't change, since we're using 2.80 of cdk below.

Reply:
Yeah, that's right. I will update this.